templates/security/passLink.html.twig

```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Bipbip Mobile</title>
+    <link href="https://fonts.googleapis.com/css?family=Century+Gothic" rel="stylesheet" />
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+            background-color: #f3f3f3;
+            font-family: Avenir Next, serif;
+        }
+
+        main {
+            font-family: Avenir Next, serif;
+            margin:30px 20px;
+        }
+
+        .header {
+            margin: 50px 0;
+        }
+
+        .email {
+            width: 100%;
+            margin: 30px 0 80px 0;
+
+        }
+    </style>
+</head>
+<body>
+    <main>
+        <div>
+
+            {% for message in app.flashes('success') %}
```

Comment:
Les 2 boucles for sont déjà dans la boucle. Ce twig étant ton squelette de mail, je ne pense pas que tu en aies besoin ici. (sauf erreur de ma part)

Reply:
Okay